packages/rps/src/core/channel-state.ts

```diff
@@ -8,8 +8,8 @@ export interface ChannelState<T = AppData> {
   status: ChannelStatus;
   aUserId: string;
   bUserId: string;
-  aDestination: string;
-  bDestination: string;
+  aAddress: string;
+  bAddress: string;
```

Comment:
We discussed that we aren't (yet) doing full justice to signing addresses vs beneficiary addresses.